homeassistant/components/shelly/light.py

```diff
@@ -130,20 +130,24 @@ def brightness(self) -> int:
         return round(255 * brightness_pct / 100)
 
     @property
-    def white_value(self) -> int:
-        """White value of light."""
-        if self.control_result:
-            white = self.control_result["white"]
-        else:
-            white = self.block.white
-        return int(white)
+    def color_mode(self) -> str | None:
+        """Return the color mode of the light."""
+        if self.mode == "color":
+            if hasattr(self.block, "white"):
+                return COLOR_MODE_RGBW
+            return COLOR_MODE_RGB
 
-    @property
-    def hs_color(self) -> tuple[float, float]:
-        """Return the hue and saturation color value of light."""
-        if self.mode == "white":
-            return color_RGB_to_hs(255, 255, 255)
+        if hasattr(self.block, "colorTemp"):
+            return COLOR_MODE_COLOR_TEMP
```

Comment:
This implies there's no shelly light with support for both color temperature and color?
If that's the case, drop the `self._supported_color_modes` member, and do like this;
```py
    @property
    def supported_color_modes(self) -> set | None:
        """Flag supported color modes."""
        return {self.color_mode}
```

Reply:
Offline discussion, `self.mode` is set by them mode of the device so it is not static